### DISCUSSION TITLE
How to run airflow on pycharm?

### DISCUSSION THREAD

**Discussion by cjj2010** [apache/airflow#39330] [Q&A]
when I execute airflow webserver on pycharm will appear build error
```

Traceback (most recent call last):
  File "/home/bjj/disk/work/airflow/airflow/cli/cli_parser.py", line 76, in <module>
    auth_mgr = get_auth_manager_cls()
  File "/home/bjj/disk/work/airflow/airflow/www/extensions/init_auth_manager.py", line 37, in get_auth_manager_cls
    auth_manager_cls = conf.getimport(section="core", key="auth_manager")
  File "/home/bjj/disk/work/airflow/airflow/configuration.py", line 1217, in getimport
    raise AirflowConfigException(
airflow.exceptions.AirflowConfigException: The object could not be loaded. Please check "auth_manager" key in "core" section. Current value: "airflow.providers.fab.auth_manager.fab_auth_manager.FabAuthManager".

```
What should I do？

**Comment by Taragolis** [comment]
Could you provide a bit more information, without it unlikely someone could help you

- How you run it? In PyCharm terminal or via Run Configurations
- Which parameters you provided to the [`airflow webserver`](https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#webserver) cli command?
- What version of Airflow?
- Are you use virtualenv or run in system Python?
- Are you setup `AIRFLOW_HOME` environemnt variable first?
- Did you regenerate `${AIRFLOW_HOME}/webserver_config.py` if you upgrade from the previous version of Airflow?

**Reply by cjj2010** [reply]
Is there a build compilation document that can help developers get started quickly？
